charts/app-config/values-production.yaml

```diff
@@ -1937,6 +1937,9 @@ govukApplications:
         - name: mail-fetcher
           command: "script/mail_fetcher"
           schedule: "*/5 * * * *"
+        - name: publish-missed-scheduled-editions
+          task: "editions:publish_missed_scheduled_editions"
+          schedule: "15 2 * * * *"
```

Comment:
What's the reason that production is scheduled to run at 0215, and integration & staging run at 5.15?

Reply:
We want to run the job on production not too long after midnight, as this when publications are often scheduled and we do not want to have much of delay. For integration and staging we want to run them after the nightly environment sync jobs, so they reflect prod accordingly.